Xwt.WPF/Xwt.WPFBackend/WidgetBackend.cs

```diff
@@ -648,6 +648,7 @@ void WidgetMouseDownHandler (object o, MouseButtonEventArgs e)
 		void WidgetMouseUpHandler (object o, MouseButtonEventArgs e)
 		{
 			var args = e.ToXwtButtonArgs (Widget);
+			args.ShouldShowContextMenu = e.ChangedButton == MouseButton.Right;

```

Comment:
Can we make `e.ToXwtButtonArgs` set the `ShouldShowContextMenu` property?

Reply:
I guess so

Reply:
@alanmcgovern, @slluis, done.